### ISSUE TITLE
Unable to Use Tuist with Cocoa pods 

### ISSUE THREAD

**Issue by rahulgoelsharechat** [tuist/tuist#3966]
@danyf90 @pietrocaselani
Problem : Need to integrate Tuist in iOS app having Cocoa pods.

**### Is It really possible to use tuist with projects with CocoaPods Yes Or No.**

I have already explored all the suggestions in here and none are working.

Current State:
I am able to generate xcodeproj with workspace using tuist

`tuist generate && pod install`

Pod installation is done but can not see the same in .workspace as separate target.
Could you please help us in solving the same with **exact** steps may it be manual or automated.

**Comment by fortmarek** [comment]
Hey @rahulgoelsharechat 👋 

> Pod installation is done but can not see the same in .workspace as separate target.

Could you rephrase this? I am not sure I follow. But yes, `tuist generate && pod install` is the right way to go about this. `tuist generate` will generate a valid Xcode project (as if you worked on it manually) and cocoapods should create the workspace with `Pods.xcodeproj`. There is nothing special we are doing with Cocoapods and Cocoapods work on top of `.xcodeproj`, so it definitely should be possible to make Cocoapods work with tuist. Adding some details describing exactly what does not work for you would be helpful here 🙏  

If you were able to create a reproducible sample, that'd be of great help, too 🙂

**Comment by VijayRaghav953** [comment]
Hey @fortmarek 
I have been working with @rahulgoelsharechat 
The exact issue is that when the app.workspace is generated and opened through tuist focus command then pods.xcproj is not linked with app.workspace , same as when install pods and open workspace.

because of that the issue we are facing is installed pods are not available for usage.
`"No such module <Pods>"`

**Comment by VijayRaghav953** [comment]
Hey @fortmarek
I just want to understand how we can link that pods.xcproj target with the app.workspace
same has been doing in cocoapods.

**Comment by danyf90** [comment]
tuist focus doesn't know about CocoaPods, so you still need to run pod install afterwards.
But probably then the podfile doesn't know about the missing targets and it will make it fail, so you probably need to keep using generate instead of focus

**Comment by VijayRaghav953** [comment]
@danyf90 
Thanks for the answer.
I understand the issue the exact situation now.
